tox.ini

```diff
@@ -82,6 +85,41 @@ commands =
     bash -c 'coverage run --source={envsitepackagesdir}/pudl -- $(which sphinx-build) -W -b html docs docs/_build/html'
     coverage xml
 
+###########################################################################
+# Continuous integration tests.
+# 
+# The following steps are run in sequence as part of the CI workflows on
+# github to run the ETL and integration tests in a modular fashion.
+###########################################################################
+[testenv:ci_ferc_to_sqlite]
```

Comment:
Let's make these environment names specific to github since that's the only place they are supposed to be run, and they can have destructive consequences for the user's environment.  "CI" is ambiguous within our vocabulary -- the main Tox environment has always been named `ci` and we don't necessarily think of CI as *only* running remotely.

```
[testenv:github_ferc_to_sqlite]
```

Reply:
For all of the `github_*` environments, can we make sure that they only run if the `$GITHUB_ACTIONS` environment variable is set, to avoid unexpected destructive side-effects within the user's `$DAGSTER_HOME`? `$GITHUB_ACTIONS` is already automatically set on the runners. I.e. if someone is fooling around locally trying to run these tox test environments they won't run, unless they go out of their way to set a `$GITHUB_ACTIONS` variably locally.

Reply:
These requests are now addressed.